hw/misc/hermes.c

```diff
@@ -1102,6 +1244,15 @@ static void pci_hermes_realize(PCIDevice *pdev, Error **errp)
     for (i = 0; i < C2H_CHANNELS; i++) {
         hermes_bar2_init_dir_reg(&hermes->bar2.c2h[i]);
     }
+
+    for (i = 0; i < UBPF_ENGINES; i++) {
+        hermes->engines[i] = ubpf_create();
+        memcpy(&bar0_init + 0x2000 + i, &zero, 1);
+
+        if (!hermes->engines[i]) {
+            error_report("Failed to create uBPF engine %i", i);
```

Comment:
I don't think we want to just report that it failed, we probably want QEMU to abort. We can do that with error_propagate().